Public/Src/Engine/Scheduler/ApiServer.cs

```diff
@@ -239,9 +239,14 @@ private async Task<IIpcResult> ExecuteGetSealedDirectoryContent(GetSealedDirecto
 
             if (failedResults.Count > 0)
             {
-                new IpcResult(
+                return new IpcResult(

```

Comment:
>return  [](start = 16, length = 7)

Oh boy. I guess this is why we didn't see the error from drop daemon? :cry:

Reply:
yeah